Diffusion.xcodeproj/project.pbxproj

```diff
@@ -963,9 +963,9 @@
 /* End XCConfigurationList section */
 
 /* Begin XCRemoteSwiftPackageReference section */
-		EB25B3D42A3A2DC4000E25A1 /* XCRemoteSwiftPackageReference "ml-stable-diffusion" */ = {
+		16AFDD4D2C1B7D4800536A62 /* XCRemoteSwiftPackageReference "ml-stable-diffusion" */ = {
 			isa = XCRemoteSwiftPackageReference;
-			repositoryURL = "https://github.com/apple/ml-stable-diffusion";
+			repositoryURL = "https://github.com/argmaxinc/ml-stable-diffusion.git";
```

Comment:
Any feedback on https://github.com/apple/ml-stable-diffusion/pull/329?

Reply:
Apple appears to still be reviewing but I'll be happy to update when that is merged - this is just for testing and review purposes here.